src/mixins/document-mixin.ts

```diff
@@ -2,6 +2,8 @@ import { Component, Vue } from 'vue-property-decorator'
 import { AxiosResponse } from 'axios'
 import { AxiosInstance as axios } from '@/utils'
 import { DocumentUpload, PdfInfoIF } from '@/interfaces'
+import * as pdfjs from 'pdfjs-dist/legacy/build/pdf'
```

Comment:
Looks like you solved the ticket I was assigned - thank you

Reply:
Not yet Travis. This works well on my local however I found out that there's an issue with the Content-Security-Policy. I need to add something there so that the script loads in Dev. Working on that now.